### PR TITLE
Correct instruction for cobbler buildiso command

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

The users are instructed to use the distro or profile names used by Cobbler in the `cobbler buildiso` command.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3
- [x] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/19619

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- In Client Config Guide added instruction for Cobbler to use the correct
```
Comment (0rnela) [inline .changelog:7]:
```suggestion
- Added instruction for Cobbler to use the correct n Client Config Guide
```

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Added instruction for Cobbler to use the correct n Client Config Guide
```
Comment (0rnela) [inline .changelog:7]:
```suggestion
- Added instruction for Cobbler to use the correct in Client Config Guide
```

File: .changelog [hunk 3]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Added instruction for Cobbler to use the correct in Client Config Guide
```
Comment (0rnela) [inline .changelog:7]:
```suggestion
- Added instruction for Cobbler to use the correct label in Client Config Guide
```

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/autoinst-cdrom.adoc [hunk 4]
```diff
@@ -45,7 +51,7 @@ For example:
 
 ----
 cobbler buildiso --systems="system1 system2 system3" \
-  --profiles="profile1 profile2 profile3 --distro=SLE_15-sp1"
+  --profiles="<your-profile1> <your-profile2> <your-profile3> --distro=<your-distro-label>"
```
Comment (meaksh) [inline modules/client-configuration/pages/autoinst-cdrom.adoc:54]: I would use here:

```suggestion
  --profiles="<your-profile1-label> <your-profile2-label> <your-profile3-label> --distro=<your-distro-label>"
```

File: modules/client-configuration/pages/autoinst-cdrom.adoc [hunk 5]
```diff
@@ -32,11 +32,17 @@ For information about creating a distribution, see xref:client-configuration:aut
...
 The Cobbler [command]``buildiso`` command takes parameters to define the name and output location of the boot ISO.
-Specifying the distribution with [option]``--distro`` is mandatory.
-For example:
+Specifying the distribution with [option]``--distro`` is mandatory when running [command]``buildiso`` command.
+
+----
+cobbler buildiso --iso=/path/to/boot.iso --distro=<your-distro-label>
+----
+
+To list the names of distributions and profiles stored by Cobbler, run the commands:
```
Comment (meaksh) [inline modules/client-configuration/pages/autoinst-cdrom.adoc:41]: I would clearly mention that you must use distro and profile labels as listed by Cobbler (not by our UI), as this was confusing for customers.

Maybe something like:

```suggestion

NOTE: Keep in mind that you must use distro and profile labels as listed by Cobbler, and not simply as shown in the UI.

To list the names of distributions and profiles stored by Cobbler, run the commands:
```

File: modules/client-configuration/pages/autoinst-cdrom.adoc [hunk 6]
```diff
@@ -32,11 +32,20 @@ For information about creating a distribution, see xref:client-configuration:aut
...
-Specifying the distribution with [option]``--distro`` is mandatory.
-For example:
+Specifying the distribution with [option]``--distro`` is mandatory when running [command]``buildiso`` command.
+
+----
+cobbler buildiso --iso=/path/to/boot.iso --distro=<your-distro-label>
+----
+
+
+NOTE: Keep in mind that you must use distro and profile labels as listed by Cobbler, and not simply as shown in the UI.
```
Comment (0rnela) [inline modules/client-configuration/pages/autoinst-cdrom.adoc:42]:
```
[IMPORTANT]
====
You must use distro and profile labels as listed by Cobbler, and not simply as shown in the UI.
====
```

**Comment by 0rnela** [comment]
Replaced with https://github.com/uyuni-project/uyuni-docs/pull/2171 (merge conflict).

**Pull request closed without merging**